### ISSUE TITLE
Cannot resize text within the Galasa run editor

### ISSUE THREAD

**Issue by hobbit1983** [galasa-dev/projectmanagement#689]
For accessibility we should be able to resize to 200% without loss of function

**Comment by hobbit1983** [comment]
This can be done by changing default text size. Cancelling
